docs/design/coordination.md

```
@@ -0,0 +1,524 @@
+|- Agent A: src/auth/     (worktree-A)
+|- Agent B: src/api/      (worktree-B)
+└- Agent C: tests/        (worktree-C)
+
+Each in isolated git worktree
+        |
+On completion: sequential merge
+|- Merge A -> main
+|- Rebase B on main, merge
+└- Rebase C on main, merge
+        |
+Textual conflicts: git detects, escalate to human or review agent
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)